services/simplified_crontab

```diff
@@ -21,9 +21,26 @@ HOME=/var/www/circulation
 # the collections.
 #
 */20 * * * * root core/bin/run bibliographic_coverage >> /var/log/cron.log 2>&1
-*/10 * * * * root core/bin/run metadata_wrangler_coverage >> /var/log/cron.log 2>&1
 30 3 * * * root core/bin/run update_nyt_bestseller_lists >> /var/log/cron.log 2>&1
 
+# Add newly discovered identifiers to our Metadata Wrangler collection.
+*/10 * * * * root core/bin/run metadata_wrangler_collection_sync >> /var/log/cron.log 2>&1
+
+# Remove newly removed identifiers from our Metadata Wrangler collection
+* */22 * * * root core/bin/run metadata_wrangler_collection_reaper >> /var/log/cron.log 2>&1
```

Comment:
The first `*` here should be `0`, and I don't know if you need the `*/` for the hourly option. Are you trying to say every day at 10pm or every 22 hours?

Reply:
I'm trying to say every 22 hours. Since these scripts might have a big performance hit on the metadata wrangler I don't want every site running them at the same time. Scheduling it in terms of the time the machine was first booted up will at least spread it out a little.

I investigated different ways of adding a random component to a crontab and didn't come up with anything that seemed like it would work without a lot of testing, but ideally I would schedule these for "once a day, after business hours"

Reply:
Okie doke, thanks for explaining. Still need the 0, then I'll approve.